earth_enterprise/rpms/build.gradle

```diff
@@ -236,10 +257,25 @@ task openGeeCommonRpm(type: Rpm, dependsOn: openGeeSharedFiles) {
         'OpenSSL and ' +
         'Skia 5.1.2'
     type = BINARY
+    autoFindProvides = true
+    autoFindRequires = true
 
     requires('opengee-postgis', '1.5.8', GREATER | EQUAL)
```

Comment:
This will be an issue with the PostGIS upgrade, along with the conflict below. Is this something that can be addressed here or will we need a separate PR on the postgres_upgrade branch?

Reply:
What is the issue and the conflict below?

If PostGIS is unbundled, we should be able to just remove this dependency.  If it's not unbundled on `master`, we should do that when we have a pull request that will unbundle it.

Reply:
It's not yet unbundled, but it is upgraded. Although, during my testing I didn't have any issues actually running it. I think it may just be reporting the wrong version. I believe the work done for upgrading it may have taken place before this was merged to master. I can update the existing PR #638 to update the version number on the RPM package, or I can create another subtask to handle that separately.

Reply:
I still see `postgis-1.5.8.tar.gz` on `master`.

Reply:
Right, I've agreed that the change doesn't need to go in this PR. It can go in a separate PR to the postgres_upgrade branch.